### ISSUE TITLE
chruby mruby failed

### ISSUE THREAD

**Issue by JuanitoFatas** [postmodern/chruby#368]
I installed mruby via `ruby-install mruby 1.2.0`, then I try to switch to mruby 1.2.0 in a new Terminal session:

```
$ chruby mruby
trace:
    [0] -:1
-:1: undefined method 'defined?' for main (NoMethodError)
```

Any more information you need? 

Thanks!

**Comment by JuanitoFatas** [comment]
However the changed is successful:

```
$ chruby mruby
trace:
	[0] -:1
-:1: undefined method 'defined?' for main (NoMethodError)
$ ruby -v
mruby 1.2.0 (2015-11-17)
^C
$ irb
mirb - Embeddable Interactive Ruby Shell

> exit
```

**Comment by steakknife** [comment]
Fixed in #328
